requirements/edx/base.txt

```diff
@@ -219,4 +219,4 @@ edx-sga==0.6.2
 redis==2.10.6
 
 # Completion
-edx-completion==0.1.7
+git+https://github.com/edx/completion@cliff/atomic-submit#egg=edx-completion==0.1.8a1
```

Comment:
My only worry here is that when https://github.com/edx/completion/pull/28 merged, this branch could be deleted?  So instead, could you push your branch to a name that will be preserved, and use the commit hash instead?

Reply:
Makes sense.  Tagging as `oc-0.1.9-1`.